### ISSUE TITLE
qt5: Error message on startup: "Unknown error code LF"

### ISSUE THREAD

**Issue by M-Reimer** [openstreetmap/merkaartor#7]
Directly after starting up Merkaartor (qt5 version) I get a popup message with an error.

![error_preferences](https://cloud.githubusercontent.com/assets/3519517/5779260/af9f7f16-9da1-11e4-8ee9-af632953e7fd.png)

On console I see the following:
```
Requesting preferences from OSM server. 

libpng warning: iCCP: too many profiles
libpng warning: iCCP: too many profiles
Received response with code  6 
```

**Comment by Krakonos** [comment]
Thanks, the error 6 is SSL; I'm working on a fix, in the meantime, you can use the API without SSL (by changing https:// to http:// in the preferences->data).

**Comment by Krakonos** [comment]
Fixed in 5ce4961.
